lib/util/merge-confidence/index.ts

```diff
@@ -25,33 +24,21 @@ export const confidenceLevels: Record<MergeConfidence, number> = {
   'very high': 2,
 };
 
-export function initConfig(): void {
-  apiBaseUrl = getApiBaseUrl();
+export function initConfig({
+  mergeConfidenceEndpoint,
+  mergeConfidenceDatasources,
+}: AllConfig): void {
+  apiBaseUrl = getApiBaseUrl(mergeConfidenceEndpoint);
   token = getApiToken();
-  supportedDatasources = parseSupportedDatasourceString();
+
+  supportedDatasources =
+    mergeConfidenceDatasources ?? presetSupportedDatasources;
```

Comment:
`parseSupportedDatasourceString()` removed as nothing is being parsed as we no longer use the experimental env vars.